languages/ruby/exercises/concept/numbers/.meta/design.md

```diff
@@ -0,0 +1,44 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student how the concept of numbers is implemented in Ruby. It will introduce this concept through the two most common numeric types in Ruby: [`Integer`][integer-ruby] (whole number) and [`Float`][float-ruby] (floating-point number).
+
+## Learning objectives
+
+- Know of the existence of the two most commonly used number types, `Integer` and `Float`.
+- Understand that an `Integer` represents whole numbers, and a `Float` represents floating-point numbers.
+- Know of basic operators such as multiplication, comparison and equality.
+- Know how to conditionally execute code using an `if`, `unless` or `case` statement.
+
+## Out of scope
+
+- Any other numeric types besides `Integer` and `Float` (so no `Rational`, `BigDecimal`, etc.).
+- Parsing an instance of `String` to an instance of `Integer` or `Float`.
+- Converting an instance of `Integer` or `Float` to an instance of `String`.
```

Comment:
```suggestion
- Coercing an instance of `Integer` or `Float` to an instance of `String`.
```

Reply:
> Coercing 

Is there another word we can use here? As a non-native speaker, "Coercing" confuses me.

Reply:
coerce: "to cause to do through pressure or necessity" - wordnet as obtained by doing `wordnet 'coerce' -s -over`

Coerce means what would be found in an English dictionary.  For example, it is never an integers desire (or character) to be sorted, yet we can coerce the behavior through `34523.to_s.chars.sort.to_i` to obtain `12345`, yet it is not converted, as `34523` remains what it was.  In this string of messages, the coercion happens first with the `to_s` call, and then coercing string to an array via `chars`, and then coercing an array to an integer.  Nothing converted, though.  Nothing casted, I can't think of another word.

One way that we might consider this would be a Person object.

Person has a name
Person has an age

In order to alphabetically sort more than one Person we must coerce that person to behave like letters do.

In order to sell restricted products, we must compare that person to an age (or number) but we don't convert the person to a number, we coerce that Person to behave like a number.  `Person > 18 && vote`

No conversion, only coercion.

Reply:
We neither convert nor coerce to strings in this exercise, so I think we can remove this line.

Reply:
For what it's worth I think "coerce" is a commonly used word, but I'm not sure if it's a programming word (which we should teach) or an english word (which we shouldn't presume knowledge of). I'd need to Google for that.

Also, I think as `Numeric#coerce` is a method that doesn't do what @kotp is suggesting, it might not be best to use it (https://ruby-doc.org/core-2.5.0/Numeric.html#method-i-coerce).

Reply:
You're right, convert is incorrect. I know it is the wrong word that's why I [brought it up](https://github.com/exercism/v3/pull/1295#discussion_r411553291) when I posted the exercise, but I couldn't find a better word. I agree with Erik.As a non-native English speaker, `coerce` confuses me also. I think we could either link to an explanation about what it means that also explains why it's not really converting or somehow avoid it. But then again, does the student need to be aware of the difference between `coercing` and `converting` at this point of the track?

Reply:
> We neither convert nor coerce to strings in this exercise, so I think we can remove this line.

It's part of the "Out of scope" section, I forked it from `c#` and it was already there, should I remove it ?

Reply:
> It's part of the "Out of scope" section

I missed that. Sorry.

I don't think the PR needs to get blocked on this. It's not a user-facing bit of the code, so let's change it to `coerce` for now as that's a "correct" word and other maintainers can always ask.

Reply:
> Also, I think as Numeric#coerce is a method that doesn't do what @kotp is suggesting, it might not be best to use it (https://ruby-doc.org/core-2.5.0/Numeric.html#method-i-coerce).

I purposefully did not mention the `Numeric#coerce`. though strictly speaking it has the features of coercion, I have never used it and do not know why it is there.

I also mention `to_s` only because this is where coercion commonly happens (without thinking about it),  Yes, we don't use it, and so, but definitely if we don't coerce these to string in the exercise, not sure why this specifically is here.

I agree (I believe I did a "comment" rather than a "change requested" but might have clicked the wrong thing.)